admin/templates/register.html

```diff
@@ -0,0 +1,73 @@
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>OSFAdmin | Log in</title>
+    <!-- Tell the browser to be responsive to screen width -->
+    <meta content='width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no' name='viewport'>
+    <!-- Bootstrap 3.3.4 -->
+    <link href="{% static 'css/bootstrap.min.css' %}" rel="stylesheet" type="text/css" />
+    <!-- Font Awesome Icons -->
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css" rel="stylesheet" type="text/css" />
+    <!-- Theme style -->
+    <link href="{% static 'css/AdminLTE.min.css' %}" rel="stylesheet" type="text/css" />
+
+    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
+    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+    <!--[if lt IE 9]>
+        <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+        <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+    <![endif]-->
+  </head>
+ <body class="register-page">
+  <div class="register-box">
+    <div class="register-logo">
+      <b>OSF</b>Admin
+    </div>
+    <div class="register-box-body">
+      <p class="login-box-msg">Register a new account</p>
+      <form action="" method="post">
+      {% csrf_token %}
+       <div class="form-group">
+          <label for="email">Email:</label>
+          <p>{{ form.email }}</p>
+          {% if form.email.errors %}<span class="text-danger"> {{ form.email.errors }}</span>{% endif %}
+      </div>
+      <div class="form-group">
+          <label for="first_name">First name:</label>
+          <p>{{ form.first_name }}</p>
+          {% if form.first_name.errors %}<span class="text-danger"> {{ form.first_name.errors }}</span>{% endif %}
+      </div>
+      <div class="form-group">
+          <label for="last_name">Last name:</label>
+          <p>{{ form.last_name }}</p>
+          {% if form.last_name.errors %}<span class="text-danger"> {{ form.last_name.errors }}</span>{% endif %}
+      </div>
+       <div class="form-group">
+          <label for="password">Password:</label>
+          <p>{{ form.password }}</p>
+          {% if form.password.errors %}<span class="text-danger"> {{ form.password.errors }}</span>{% endif %}
+      </div>
+      <!-- include widget for displaying multi-checkbox permission list -->
+        <div class="row">
+          <div class="col-xs-4">
+            <button type="submit" class="btn btn-primary btn-block btn-flat">Register</button>
+          </div><!-- /.col -->
+        </div>
+      </form>
+
+      <a href="{% url 'auth:login' %}" class="text-center">I already have an account</a>
+    </div><!-- /.form-box -->
+  </div><!-- /.register-box -->
+
+   <!-- jQuery 2.1.4 -->
+  <script src="{% static 'js/jquery.min.js' %}"></script>
+  <!-- Bootstrap 3.3.2 JS -->
+  {% render_bundle 'vendor' %}
+  <!-- AdminLTE App -->
+  {% render_bundle 'admin-base-page' %}
```

Comment:
I had to do this for the last PR too and I realized I didn't note it. This bundle doesn't exist anymore. By changing admin-base-page to app.min I got pages to load again. Nothing like a tightly-coupled asset framework to give you a hard 500 error if your assets aren't in the right place.